ee-9/feature-pack/pom.xml

```diff
@@ -440,7 +440,31 @@
             <scope>provided</scope>
         </dependency>
 
-        <!-- module and copy artifact dependencies -->
+	<!-- module and copy artifact dependencies -->
+
+           <dependency>
+                <groupId>org.antlr</groupId>
+                <artifactId>antlr4</artifactId>
+		<scope>provided</scope>
+                <exclusions>
+                    <exclusion>
+                        <groupId>*</groupId>
+                        <artifactId>*</artifactId>
+                    </exclusion>
+                </exclusions>
```

Comment:
Please drop the exclusions sections from all of these. The dependencyManagement additions in ee-9/pom.xml already handle this.

Reply:
Addressed the ones I added.